src/libraries/System.Formats.Cbor/src/CompatibilitySuppressions.xml

```diff
@@ -1,17 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Deleting this file is failing the package validation in CI:

> 

.packages\microsoft.dotnet.compatibility\2.0.0-preview.4.22252.4\build\Microsoft.NET.Compatibility.Common.targets(33,5): error CP0002: (NETCORE_ENGINEERING_TELEMETRY=Build) Member 'System.Formats.Cbor.CborReader.ReadHalf()' exists on [Baseline] lib/net5.0/System.Formats.Cbor.dll but not on lib/netstandard2.0/System.Formats.Cbor.dll

Going by https://github.com/dotnet/runtime/pull/62872#issuecomment-1009263851, it looks like we need to keep this file. Please revert its deletion (`git checkout dotnet/main src/libraries/System.Formats.Cbor/src/CompatibilitySuppressions.xml`).

Reply:
Done. 🌹